lib/block_view_post.go

```diff
@@ -755,6 +756,13 @@ func (bav *UtxoView) _connectSubmitPost(
 		copy(repostedPostHash[:], repostedPostHashBytes)
 		delete(extraData, RepostedPostHash)
 	}
+	isFrozen := false
+	if blockHeight >= bav.Params.ForkHeights.AllowUpdatingNFTPostsBlockHeight {
+		if extraDataIsFrozen, exists := extraData[IsFrozen]; exists {
+			isFrozen = bytes.Equal(extraDataIsFrozen, []byte{1})
```

Comment:
we can mimic what we did with `isQuotedRepost` above. we should define `var FrozenPostVal = []byte{1}` in constants.go so we can consistently reference it everywhere. I do prefer using bytes.Equal tho as you have it here as opposed to reflect.DeepEqual